<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATCDetailsWindow</class>
 <widget class="QWidget" name="ATCDetailsWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATC details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imgs.qrc">
    <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="CallsignLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FacilityLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="InfoLabel" name="NameLabel" native="true"/>
     </item>
     <item>
      <widget class="InfoLabel" name="FrequencyLabel" native="true"/>
     </item>
     <item>
      <widget class="InfoLabel" name="RatingLabel" native="true"/>
     </item>
     <item>
      <widget class="InfoLabel" name="AirportLabel" native="true"/>
     </item>
     <item>
      <widget class="InfoLabel" name="ServerLabel" native="true"/>
     </item>
     <item>
      <widget class="InfoLabel" name="TimeOnlineLabel" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="AtisMessageLabel">
       <property name="text">
        <string>Atis message</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="AtisMessageField">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ShowButton">
       <property name="text">
        <string>Show</string>
       </property>
       <property name="icon">
        <iconset theme="go-jump"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoLabel</class>
   <extends>QWidget</extends>
   <header>ui/widgets/infolabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>ATCDetailsWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
